inc/Engine/Optimization/RUCSS/Database/Tables/UsedCSS.php

```diff
@@ -75,6 +79,7 @@ protected function set_schema() {
 			KEY modified (modified),
 			KEY last_accessed (last_accessed),
 			INDEX `status_index` (`status`(191)),
+			INDEX `error_code_index` (`error_code`(191)),
```

Comment:
```suggestion
			INDEX `error_code_index` (`error_code`),
```

we don't need to worry about this 191 utf8 and multibyte issue because the column size is already less than 191.